docs/howto/operate/grafana.md

```diff
@@ -50,6 +86,7 @@ Note this filepath is _relative_ to the location of your `cluster.yaml` file.
 support:
   helm_chart_values_files:
     - support.values.yaml
+    - enc-support.secret.yaml
```

Comment:
TODO: Systematic rename of this file to `enc-support.secret.values.yaml`

Reply:
Just done.